docker-compose.yaml

```diff
@@ -29,10 +29,20 @@ services:
       ELIGIBILITY_API_URI: http://host.docker.internal:3010/api 
       APPLICATION_API_URI: http://host.docker.internal:3001/api
       DEFRA_ID_ENABLED: ${DEFRA_ID_ENABLED:-false}
-      DEFRA_ID_TENANT: ${DEFRA_ID_TENANT:-changeme}
-      DEFRA_ID_POLICY: ${DEFRA_ID_POLICY:-changeme}
+      DEFRA_ID_TENANT: ${DEFRA_ID_TENANT:-azdcuspoc5}
+      DEFRA_ID_POLICY: ${DEFRA_ID_POLICY:-B2C_1A_SIGNUPSIGNINSFI}
       DEFRA_ID_REDIRECT_URI: ${DEFRA_ID_REDIRECT_URI:-http://localhost:3000/apply/signin-oidc}
-      DEFRA_ID_CLIENT_ID: ${DEFRA_ID_CLIENT_ID:-change_me}
-      DEFRA_ID_CLIENT_SECRET: ${DEFRA_ID_CLIENT_SECRET:-change_me}
-      DEFRA_ID_JWT_ISSUER_ID: ${DEFRA_ID_JWT_ISSUER_ID:-change_me}
-      DEFRA_ID_SERVICE_ID: ${DEFRA_ID_SERVICE_ID:-changeme}
\ No newline at end of file
+      DEFRA_ID_JWT_ISSUER_ID: ${DEFRA_ID_JWT_ISSUER_ID:-64c9d4f5-a560-4b65-9004-6d1e5ccee51d}
+      DEFRA_ID_CLIENT_ID: ${DEFRA_ID_CLIENT_ID:-83d2b160-74ce-4356-9709-3f8da7868e35}
```

Comment:
Several DEFRA ID values changed from 'changeme' to actual values, is this intended?